sleap/gui/dataviews.py

```diff
@@ -721,3 +721,11 @@ def get_item_color(self, instance_group: InstanceGroup, key: str):
             # Get the InstanceGroup and color
             color = color_manager.get_instance_group_color(instance_group, frame_group)
             return QtGui.QColor(*color)
+    
+    #TODO: Add methods to allow the name property of an instance group to be editable in the table view.
```

Comment:
Remove the TODO since you have already done it. You can leave the rest of the comment behind, but not really necessary.

```suggestion
```